db/models/score/scores.py

```diff
@@ -0,0 +1,41 @@
+"""The module containing all code related to the `scores` table
+within the Postgres db.
+"""
+import uuid
+from db import db
+from sqlalchemy import ForeignKey
+from sqlalchemy.sql import func
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class JustScore(db.Model):
```

Comment:
⛏️ Make model and table name clearer - ie don't abbreviate.

Reply:
have changed from JustScore to Score 👍🏾